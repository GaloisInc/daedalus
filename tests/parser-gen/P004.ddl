
def Main = { $$ = Many (4..5) (Match1 ('0'..'7')) ; END }
