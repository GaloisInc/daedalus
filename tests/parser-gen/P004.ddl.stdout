module P004
 
--- Imports:
 
--- Type defs:
 
--- Rules:
 
P004.Main : Grammar [uint 8] =
  do ($$ : [uint 8]) <- Many[ 4 .. 5 ] Match ('0' .. '7')
     END
     pure $$
 
P004.Main_2 : Grammar {} =
  do @Many[ 4 .. 5 ] @Match ('0' .. '7')
     END
